lang/ilias_hu.lang

```diff
@@ -13404,7 +13404,6 @@ rbac#:#cadm_edit_permission#:#A felhasználó módosíthatja a Rendszerbeállít
 common#:#no_gallery_users_available#:#Jelenleg egy felhasználó sincs a képtárban.
 crs#:#crs_mem_contact#:#Technikai segítségnyújtó
 awrn#:#awareness_now_online#:#Jelenleg online
-awrn#:#awareness_now_online_users#:#[ONLINE_USER_NAMES]
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.